test/abstractmcmc.jl

```diff
@@ -10,7 +10,7 @@ include("common.jl")
 
     θ_init = randn(rng, 2)
 
-    model = AdvancedHMC.DifferentiableDensityModel(ℓπ_gdemo, ForwardDiff)
+    model = AdvancedHMC.LogDensityModel(ℓπ_gdemo, ForwardDiff)
```

Comment:
This should be
```suggestion
    model = AdvancedHMC.LogDensityModel(ADgradient(Val(:forwarddiff), ℓπ_gdemo))
```
(or something similar, don't remember the API by heart), shouldn't it?

Reply:
Ah, good catch; thanks!